Source/Core/ArcGisImageServerTerrainProvider.js

```diff
@@ -200,11 +202,15 @@ define([
      * @param {Number} x The X coordinate of the tile for which to request geometry.
      * @param {Number} y The Y coordinate of the tile for which to request geometry.
      * @param {Number} level The level of the tile for which to request geometry.
+     * @param {Boolean} [throttleRequests=true] True if the number of simultaneous requests should be limited,
```

Comment:
When this goes into master - or sooner - include all changes to the public API in CHANGES.md.